### PR TITLE
update framework

### PR DESCRIPTION
**Pull request by lohanidamodar**
## What does this PR do?

- Framework with Router V2

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Screenshots may also be helpful.)

## Related PRs and Issues

- (Related PR or issue)

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [x] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

**Review by eldadfux — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -51,8 +51,8 @@
         "utopia-php/config": "0.2.*",
         "utopia-php/database": "0.38.*",
         "utopia-php/domains": "1.1.*",
-        "utopia-php/dsn": "0.1.*",
         "utopia-php/framework": "0.28.*",
```
Comment (abnegate) [inline composer.json:54]: Shouldn't this have been updated too?

File: app/controllers/api/users.php [hunk 2]
```diff
@@ -1023,41 +1023,6 @@ function createUser(string $hash, mixed $hashOptions, string $userId, ?string $e
...
 
-App::patch('/v1/users/:userId/verification')
-    ->desc('Update Email Verification')
-    ->groups(['api', 'users'])
-    ->label('event', 'users.[userId].update.verification')
-    ->label('scope', 'users.write')
-    ->label('audits.event', 'verification.update')
-    ->label('audits.resource', 'user/{request.userId}')
-    ->label('audits.userId', '{request.userId}')
-    ->label('sdk.auth', [APP_AUTH_TYPE_KEY])
```
Comment (christyjacob4) [inline app/controllers/api/users.php:1034]: don't we need this endpoint anymore ? I don't see any other endpoint where we are able to update email verification

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
Please check the comments ✅

File: app/controllers/api/users.php (hunk 2 above)
Comment (lohanidamodar) [reply app/controllers/api/users.php:1034]: It was in two places probably some merge issues.

File: tests/e2e/Services/Storage/StorageBase.php [hunk 3]
```diff
@@ -764,13 +764,6 @@ public function testDeleteBucketFile(array $data): array
         $this->assertEquals(204, $file['headers']['status-code']);
         $this->assertEmpty($file['body']);
```
Comment (abnegate) [inline tests/e2e/Services/Storage/StorageBase.php]:
```suggestion

        $file = $this->client->call(Client::METHOD_GET, '/storage/buckets/' . $data['bucketId'] . '/files/' . $data['fileId'], array_merge([
            'content-type' => 'application/json',
            'x-appwrite-project' => $this->getProject()['$id'],
        ], $this->getHeaders()));

        $this->assertEquals(404, $file['headers']['status-code']);
```

**Review by abnegate — APPROVED** [review]

**Pull request merged**